assets/src/edit-story/app/prepublish/constants.js

```diff
@@ -186,25 +186,17 @@ export const MESSAGES = {
       ),
     },
     MISSING_IMAGE_ALT_TEXT: {
-      MAIN_TEXT: __('Image missing alt text', 'web-stories'),
-      HELPER_TEXT: sprintf(
-        /* translators: %s: alt. */
-        __(
-          'Add meaningful “%s” text to images to optimize accessibility and indexability.',
-          'web-stories'
-        ),
-        'alt'
+      MAIN_TEXT: __('Image missing assistive text', 'web-stories'),
+      HELPER_TEXT: __(
+        'Add meaningful assistive text to images to optimize accessibility and indexability.',
+        'web-stories'
       ),
     },
     MISSING_VIDEO_ALT_TEXT: {
-      MAIN_TEXT: __('Video missing alt text', 'web-stories'),
-      HELPER_TEXT: sprintf(
-        /* translators: %s: alt. */
-        __(
-          'Add meaningful “%s” text to videos to optimize accessibility and indexability.',
-          'web-stories'
-        ),
-        'alt'
+      MAIN_TEXT: __('Video missing assistive text', 'web-stories'),
```

Comment:
"Video missing description"

See #5332.

Reply:
Ah, we are changing that again.